Packs/CofenseTriage/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,2 @@
+* Added support for `cofense-get-report-png-by-id` and `cofense-get-threat-indicators`
+* Internal refactoring
```

Comment:
please use the demisto-sdk command `update-release-notes` for that.
https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/update_release_notes/README.md